src/main/kotlin/view/TopBar.kt

```diff
@@ -40,92 +40,20 @@ class TopBar : View() {
                 }
             }
         }
-//        menu("_View") {
-//            item("_Basic Actions") {
-//                action {
-//
-//                }
-//            }
-//            item("_RGB")
-//            item("_HSV")
-//        }
-        menu("_Help") {
-            item("_How to")
-        }
-    }
-
-    private fun imageOperation(mode: String) {
-
-        var fileSelectorTitle = ""
-        var fileSelectorFilter = emptyArray<FileChooser.ExtensionFilter>()
-        var fileSelectorMode = FileChooserMode.None
-
-        val importFilter = arrayOf(
-            FileChooser.ExtensionFilter(
-                "PNG files (*.png)",
-                "*.png"
-            ),
-            FileChooser.ExtensionFilter(
-                "Bitmap files (*.bmp)",
-                "*.bmp"
-            ),
-            FileChooser.ExtensionFilter(
-                "JPEG files (*.jpeg, *.jpg)",
-                "*.jpeg",
-                "*.jpg"
-            )
-        )
-
-        val exportFilter = arrayOf(
-            importFilter[0], importFilter[1],
-            FileChooser.ExtensionFilter(
-                "JPEG files (*.jpg)",
-                "*.jpg"
-            )
-        )
+        menu("View") {
+            item("Basic Filters") {
+                action {
 
-        try {
-            when (mode) {
-                "import" -> {
-                    fileSelectorTitle = "Import image"
-                    fileSelectorFilter = importFilter
-                    fileSelectorMode = FileChooserMode.Single
                 }
-                "export" -> {
-                    fileSelectorTitle = "Export image"
-                    fileSelectorFilter = exportFilter
-                    fileSelectorMode = FileChooserMode.Save
-                }
-            }
-            val dir = chooseFile(
-                title = fileSelectorTitle,
-                filters = fileSelectorFilter,
-                mode = fileSelectorMode
-            ) {
-                initialDirectory = File(File("").canonicalPath)
-                initialFileName = "IPEwG_result_image"
             }
-            if (dir.isNotEmpty())
-                when (mode) {
-                    "import" ->
-                        fileController.load("file:///" + dir[0].toString())
-                    "export" ->
-                        fileController.save(
-                            dir[0].toString(),
-                            dir[0].toString().substring(
-                                dir[0].toString().lastIndexOf(".") + 1,
-                                dir[0].toString().length
-                            )
-                        )
-                }
+            item("Steganography") {
+                action {
```

Comment:
I think you are trying to allow user jump to steganography when they select in view button in the top bar? currently selecting in top bar does not have any effect

Reply:
Yeah, this should be deleted